src/psyclone/tests/domain/gocean/transformations/gocean_move_iteration_boundaries_inside_kernel_trans_test.py

```diff
@@ -206,8 +205,8 @@ def test_go_move_iteration_boundaries_inside_kernel_two_kernels_apply_twice():
   ystart_1 = 1
   ystop_1 = SIZE(uold_fld%data, 2)
   do j = 1, SIZE(uold_fld%data, 2), 1
-    do i_1 = 1, SIZE(uold_fld%data, 1), 1
-      call time_smooth_code(i_1, j, u_fld%data, unew_fld%data, uold_fld%data, \
+    do i = 1, SIZE(uold_fld%data, 1), 1
+      call time_smooth_code(i, j, u_fld%data, unew_fld%data, uold_fld%data, \
 xstart_1, xstop_1, ystart_1, ystop_1)
     enddo
   enddo
```

Comment:
Can you do away with line 218 now? Also, I can't figure out the reason for this change in the generated code - please can you explain?

Reply:
> Can you do away with line 218 now?

Yes, removed

> Also, I can't figure out the reason for this change in the generated code - please can you explain?

It's the GOLoop now declaring the variables in the InvokeSchedule instead of the inner scope. This means that `i` is shared between GOLoops, avoiding the `i_1`, `i_2`... This is not just stylistic, I realized that some transformations like moving and merging GOcean loop/kernels assume the symbol is the same (because in gen_code() the symbol is always the same), so it is necessary for correctness.

Reply:
Aha. Thanks for explaining.